tests/checksum_integration_tests.rs

```diff
@@ -6,6 +6,7 @@ use std::fs;
 use std::process::Command;
 
 #[test]
+#[cfg_attr(miri, ignore)] // Miri doesn't allow this due to isolation restrictions
```

Comment:
You can set `MIRIFLAGS=-Zmiri-disable-isolation` if you want to run such tests anyway.